src/openforms/utils/sanitizer.py

```diff
@@ -0,0 +1,190 @@
+from django.core.files.base import File
```

Comment:
```suggestion
from django.core.files import File
```
Whenever Django "re-exports" something in the package "above", that's the preferred API to import it from.

Reply:
Ah, check